src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-row-view.ts

```diff
@@ -317,53 +317,39 @@ class GmailThreadRowView {
     this._userView = userView;
   }
 
-  getCounts(): Counts {
-    let counts = this._counts;
-
-    if (!counts) {
-      const recipientsElement = querySelector(this._elements[0], 'td div.yW');
-
-      if (this._driver.isUsingSyncAPI()) {
-        const draftCount = recipientsElement.querySelectorAll('.boq').length;
-        const messageCountMatch = recipientsElement.querySelector('.bx0');
-        const messageCount =
-          messageCountMatch && messageCountMatch.innerHTML
-            ? +messageCountMatch.innerHTML
-            : draftCount
-            ? 0
-            : 1;
-        counts = this._counts = {
-          messageCount,
-          draftCount,
-        };
-      } else {
-        const [preDrafts, drafts] = recipientsElement.innerHTML.split(
```

Comment:
Based on numbers from #993, I've removed the `!this._driver.isUsingSyncAPI()` block.